src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -797,4 +746,12 @@ public void addToDocumentsGenerated(final ListValue<DivorceDocument> listValue)
             documents.add(listValue);
         }
     }
+
+    public Jurisdiction getJurisdiction() {
+        if (jurisdiction == null) {
+            jurisdiction = new Jurisdiction();
+        }
+
+        return jurisdiction;
+    }
```

Comment:
do we need this ?

Reply:
I tried to set a default in the property definition but it causes issues with lombok so this was the only way I could see if ensuring there was an object